src/libstore/sqlite.cc

```diff
@@ -58,6 +62,11 @@ SQLite::SQLite(const Path & path, bool create)
     if (sqlite3_busy_timeout(db, 60 * 60 * 1000) != SQLITE_OK)
         SQLiteError::throw_(db, "setting timeout");
 
+    if (false) {
```

Comment:
I don't profess to be a C/C++ master, but I've seen other places in the codebase where we use `#if 0` and `#endif` to guard these kinds of things, rather than  an `if (false)`. Is there a reason we don't do the same here?

To my naive understanding, the difference is that `#if 0` would not be parsed / compiled / etc at all, while `if (false)` might? Maybe compilers are smart enough to deal with that, though.

Reply:
I did try `#if 0`, but that resulted in a warning about unused `traceSQL`. I figured this was the least obtrusive solution.

I don't like the alternative of adding more machinery to take care of `traceSQL`. If we're not going minimal, I'd rather improve the logger situation so that it's more configurable at runtime.

Reply:
Gotcha, that makes sense. Thanks for trying my silly suggestion!